app/components/ingredient-list-edit.js

```diff
@@ -0,0 +1,33 @@
+import Ember from 'ember';
+
```

Comment:
In this component you don't declare constants.  I guess, it's just more  question for me to clarify for myself. Do we need to declare them in every component? Or we don't? I was expecting according to our previous code see in this file something like this:
```javascript
const {
   get,
   let
} = Ember;
```
I just want to know for myself for the future what is the rule.